src/coreclr/vm/threads.h

```diff
@@ -1245,11 +1245,7 @@ class Thread
     }
 #endif // DACCESS_COMPILE
 
-    DWORD IsExecutingOnAltStack()
-    {
-        LIMITED_METHOD_CONTRACT;
-        return (m_State & TS_ExecutingOnAltStack);
-    }
+    DWORD IsExecutingOnAltStack();
```

Comment:
I am wondering why the implementation was moved to .cpp.

Reply:
Ah, I should move it back. Initially I had this checking the thread local variables to report this, but I realized that we use during GC stack walking, and checking the current threads green thread state is a bad idea.